### PR TITLE
PrivacyProfile beta documentation

### PR DESCRIPTION
**Pull request by Gwen-Microsoft**

### CODE REVIEW

File: api-reference/beta/api/privacyprofile_get.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+Since the **privacyProfile** resource supports [extensions](../../../concepts/extensibility_overview.md), you can also use the `GET` operation to get custom properties and extension data in an **privacyProfile** instance.
+
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](../../../concepts/permissions_reference.md).
+
+
+|Permission type      | Permissions (from least to most privileged)              |
+|:--------------------|:---------------------------------------------------------|
+|Delegated (work or school account) | User.Read    |
```
Comment (dkershaw10) [inline api-reference/beta/api/privacyprofile_get.md]: Did you explicitly add viewing this property to the User.Read Permission?
Also add Directory.Read.All

File: api-reference/beta/api/privacyprofile_get.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+# Get privacyProfile
```
Comment (dkershaw10) [inline api-reference/beta/api/privacyprofile_get.md]: We don't normally add a GET/PATCH API operation for properties of an entity type.  We only API methods for entity types and functions/actions.  There's no need for this or the patch topic.  Please remove them.  Sorry you ended up creating these.
The property should just get documented on the Organization resource topic.

File: api-reference/beta/api/privacyprofile_patch.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
+# Add a privacyProfile
```
Comment (dkershaw10) [inline api-reference/beta/api/privacyprofile_patch.md]: Remove topic per the last comment.

File: api-reference/beta/resources/organization.md [hunk 4]
```diff
@@ -36,6 +36,7 @@ Only the read and update operations are supported on tenants; create and delete
 |objectType|String|A string that identifies the object type. For tenants the value is always “Company”. |
 |postalCode|String| Postal code of the address for the organization |
 |preferredLanguage|String| The preferred language for the organization. Should follow ISO 639-1 Code; for example "en". |
+|privacyProfile|[PrivacyProfile](privacyprofile.md)| The privacy profile of an organization.            |
```
Comment (dkershaw10) [inline api-reference/beta/resources/organization.md]: PrivacyProfile ->privacyProfile
You've also missed out some elements that need update.  In the **methods** section:
Update | organization | Update organization object. (Only the marketingNotificationMails and **technicalNotificationMails** properties can be updated.)

You need to add privacy privacyProfile to this list.  In testing I discovered this actually needs further update, since these can also be updated, so please add to the list.
securityComplianceNotificationMails, securityComplianceNotificationPhones,

File: api-reference/beta/resources/organization.md [hunk 5]
```diff
@@ -76,6 +77,7 @@ Here is a JSON representation of the resource
   "onPremisesSyncEnabled": true,
   "postalCode": "string",
   "preferredLanguage": "string",
+  "privacyProfile": {"@oadata.type": "microsoft.graph.privacyProfile"},
```
Comment (dkershaw10) [inline api-reference/beta/resources/organization.md]: Also I see another bug, that is in the Organization API topics...

**Get organization** - can you fix the permissions please?
|Permission type      | Permissions (from least to most privileged)              |
|:--------------------|:---------------------------------------------------------|
|Delegated (work or school account) | User.Read, Directory.Read.All |
|Delegated (personal Microsoft account) | Not supported.    |
|Application | Directory.Read.All |

**Update organization** (organization_update.md) - can you fix the permissions please?  Also more later...
|Permission type      | Permissions (from least to most privileged)              |
|:--------------------|:---------------------------------------------------------|
|Delegated (work or school account) | Directory.AccessAsUser.All  |
|Delegated (personal Microsoft account) | Not supported.    |
|Application | Not supported. |

Also the **Request body** section is wrong.  Most of those properties cannot be set AND you need to add privacyPolicy.  You should only have marketingNotificationEmails, technicalNotificationMails, privacyProfile, securityComplianceNotificationMails, and securityComplianceNotificationPhones in the table.
And you'll need to fix the example.

Basically this topic is ion a right state, and you are the lucky one to clean it up :)

File: api-reference/beta/api/organization_update.md [hunk 6]
```diff
@@ -28,34 +28,19 @@ In the request body, supply the values for relevant fields that should be update
...
-|provisioningErrors|ProvisioningError|                                        **Notes**: not nullable.            |
-|securityComplianceNotificationMails|String||
-|securityComplianceNotificationPhones|String||
-|state|String|            |
-|street|String|            |
-|technicalNotificationMails|String|                                        **Notes**: not nullable.            |
-|telephoneNumber|String|            |
-|verifiedDomains|VerifiedDomain|The collection of domains associated with this tenant.                            **Notes**: not nullable.            |
+|marketingNotificationEmails|Collection (String)|                                        **Notes**: not nullable.            |
+|privacyProfile|[PrivacyProfile](privacyprofile.md)|The privacy profile of an organization (set statementUrl and contactEmail).            |
```
Comment (dkershaw10) [inline api-reference/beta/api/organization_update.md]: Lower camel case for the type please.  This is also where your build break is:
the link should be to ../resources/privacyprofile.md

**Comment by dkershaw10** [comment]
@lleonard-msft LGTM
